### DISCUSSION TITLE
Auto-Update never seems to work?

### DISCUSSION THREAD

**Discussion by sKlat-beep** [SnosMe/awakened-poe-trade#708] [Ideas]
Maybe I just don't understand how it works... but when I "check for updates" it finds the update, then says it will be installed on exit. It doesn't seem to mean exiting POE, because that doesn't trigger the update... and if I just right-click and "quit" the application it doesn't do anything either, so when I re-open awakened poe trade it's still the old version and I can re-check for the update.

Can we just add the functionality to update immediately?

**Comment by SnosMe** [comment]
Yep, it can fail for some people. Since auto-updater is not written by me, the workaround is to restart it with admin rights and then quit or simply download from GitHub again ("auto-updated" 😆 ).
